### DISCUSSION TITLE
meow + general

### DISCUSSION THREAD

**Discussion by ChauhanT** [meow-edit/meow#203] [Ideas]
Hi, 

Has anyone tried to combine `meow` with `general.el` ? I really like how doom/spacemacs etc. organise their menus, and I have organised mine with `general.el` (and `evil`, which I am looking to drop in favour of `meow`). Is it possible? 

I know that `meow` was designed to leverage emacs native bindings rather than create new leader-key style keybindings. I'm just trying to see if I can combine `meow` with `general.el` to achieve similar menus. 

Any help/discussions/links/opinions are super appreciated.

Cheers!

EDITS: So many darn typos X(

**Comment by DogLooksGood** [comment]
AFAIK, you can use general, but there's no major-mode based leader system in meow.

**Reply by ChauhanT** [reply]
I think `general.el` does though, through its `:mode` and `:keymaps` options. So maybe it can work? I'll give it a go this weekend and let's see how it turns out.

**Reply by TheEccentricConnisure** [reply]
@ChauhanT  were you able to get it to work?

**Comment by TheEccentricConnisure** [comment]
I've installed both packages and want to leverage their strengths, but i havent been successful in the process. 

I have installed the packages in the following manner:
```
    (use-package meow
       :ensure t
       :config
  (require 'meow-qwerty)
  (meow-setup)
  (meow-global-mode 1)
  (setq meow-expand-exclude-mode-list (remove 'org meow-expand-exclude-mode-list))
  (setq meow-use-clipboard t)
  )
```
- the general.el
```
   (use-package general
     :config
     (general-evil-setup)
      (general-create-definer leader-key
          :states '(normal insert visual emacs)
          :keymaps 'override
          :prefix "SPC" ;; set leader
          :global-prefix "C-SPC");; access leader in insert mode
  (require 'hashmap)
   )

```
THe hasmap.el has all my custom keybindigs
